src/decorator/CircuitBreakerDecorator.test.ts

```diff
@@ -0,0 +1,42 @@
+import { CircuitBreakerError } from '../CircuitBreakerError';
+import { CircuitBreakerDecorator } from './CircuitBreakerDecorator';
+
+describe('CircuitBreakerDecorator test Suite', () => {
+    @CircuitBreakerDecorator({
+        failureThreshold: 1,
+        recoveryTimeout: 3000,
+    })
+    class ClassUnderTest {
+        public unstableFn(shouldThrow: boolean): boolean {
+            if (shouldThrow === true) {
+                throw new Error('I am failing');
+            }
+            return shouldThrow;
+        }
+    }
+
+    let myInstance: ClassUnderTest;
+
+    beforeEach(() => {
+        myInstance = new ClassUnderTest();
+    });
+
+    it('should throw a CircuitBreakerError when consecutive fail are superior to 1', () => {
```

Comment:
For consistency purpose, could you use the `test(...)` method rather that `it(...)` method (or change all the other tests for using `it` ?